package/MDAnalysis/analysis/pca.py

```diff
@@ -0,0 +1,353 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# MDAnalysis --- http://www.MDAnalysis.org
+# Copyright (c) 2006-2015 Naveen Michaud-Agrawal, Elizabeth J. Denning, Oliver Beckstein
+# and contributors (see AUTHORS for the full list)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+"""
+Principal Component Analysis (PCA) --- :mod:`MDAnalysis.analysis.pca`
+=====================================================================
+
+:Authors: John Detlefs
+:Year: 2016
+:Copyright: GNU Public License v3
+
+This module contains the linear dimensions reduction method Principal Component
+Analysis (PCA). PCA sorts a simulation into 3N directions of descending
+variance, with N being the number of atoms. These directions are called
+the principal components. The dimensions to be analyzed are reduced by only
+looking at a few projections of the first principal components. To run a
+Principal Component Analysis, please refer to the :ref:`PCA-tutorial`.
+
+This module constructs a covariance matrix wherein each element of the matrix
+is denoted by (i,j) row-column coordinates. The (i,j) coordinate is the
```

Comment:
`The PCA uses the covariance matrix, a 3N x 3N matrix where the element (i,j) is the covariance between coordinates i and j. `

Shorter and less potential for error in the text. Because right now you are using the word `coordinate` to donate an element in the matrix but also a normal coordinate. (I prefer to refer only of elements in a matrix context)